terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -432,6 +445,131 @@
+          dependency = subject.find { |x| x.name == "distribution_label::bitbucket::cloudposse/terraform-null-label" }
+          expect(dependency).to_not be_nil
+          expect(dependency.version).to be_nil
+          expect(dependency.requirements).to match_array([{
+            requirement: nil,
+            groups: [],
+            file: "child_module_two/main.tf",
+            source: {
+              type: "git",
+              url: "https://bitbucket.org/cloudposse/terraform-null-label.git",
+              branch: nil,
+              ref: nil
+            }
+          }])
+        end
+
+        it "has the right details for the dns_dup with duplicate git repo" do
+          dependency = subject.find do |x|
+            x.name == "dns_dup::github::cloudposse/terraform-aws-route53-cluster-zone::tags/0.2.5"
+          end
+          expect(dependency).to_not be_nil
+          expect(dependency.version).to eq("0.2.5")
+          expect(dependency.requirements).to match_array([{
+            requirement: nil,
+            groups: [],
+            file: "main.tf",
+            source: {
+              type: "git",
+              url: "https://github.com/cloudposse/terraform-aws-route53-cluster-zone.git",
+              branch: nil,
+              ref: "tags/0.2.5"
+            }
+          }])
+        end
+
+        it "has the right details for the dns with child module duplicate and duplicate git repo" do
+          dependency = subject.find do |x|
+            x.name == "dns::github::cloudposse/terraform-aws-route53-cluster-zone::tags/0.2.5"
+          end
+          expect(dependency).to_not be_nil
+          expect(dependency.version).to eq("0.2.5")
+          expect(dependency.requirements).to match_array([{
+            requirement: nil,
+            groups: [],
+            file: "child_module_two/main.tf",
+            source: {
+              type: "git",
+              url: "https://github.com/cloudposse/terraform-aws-route53-cluster-zone.git",
+              branch: nil,
+              ref: "tags/0.2.5"
+            }
+          },
+                                                          {
+                                                            requirement: nil,
+                                                            groups: [],
+                                                            file: "main.tf",
+                                                            source: {
+                                                              type: "git",
+                                                              url: "https://github.com/cloudposse/terraform-aws-route53-cluster-zone.git",
+                                                              branch: nil,
+                                                              ref: "tags/0.2.5"
+                                                            }
+                                                          }])
```

Comment:
Could you fix the spacing here?

Reply:
Fixed, I had to restructure some as that whitespace was introduced by `rubocop -a` Now I know to double check what it does for spacing :)